examples/cpp_torch_examples/mpconfigport.h

```diff
@@ -0,0 +1 @@
+#include "mpconfigport_minimal.h"
```

Comment:
just rename  mpconfigport_minimal.h to mpconfigport.h here to make it one less file.